src/lib/header-bidding/a9/a9.ts

```diff
@@ -70,11 +64,23 @@ const requestBids = async (
 		return requestQueue;
 	}
 
+	const section = window.guardian.config.page.section;
+
+	const isNetworkFront =
+		section === 'uk' || section === 'us' || section === 'au';
+
+	const filteredAdUnits = adUnits.filter((adUnit) => {
```

Comment:
We still want a9 to work in the other slots, just nut gumgum afaik.

Reply:
Yep, Jake's right. I see the issue here, we want:
- A9 to run everywhere
- But prevent GumGum running in certain slots, when it serves on section fronts

As I type this is realise it's quite the messy rule want to chat it through @dskamiotis?

Reply:
thanks @Amouzle I have an idea/solution, but will chat it through to confirm if thats ok

Reply:
This still prevents a9 from running for all the other slots on fronts, it's also preventing all bidders not just GumGum?

Reply:
I thought we only wanted specif slots for the network and section fronts?
On Network Front === inline1 (as we would like it further down the page)

On Section Fronts === top-above-nav
and all other slots for non fronts?

Reply:
We want only GumGum to be blocked in the other slots, the code you've got prevents any A9 bidder not just GumGum.
As @Amouzle we still want A9 working everywhere.